packages/utils/telemetry-utils/src/telemetryTypes.ts

```diff
@@ -70,24 +86,32 @@ export interface ITelemetryGenericEventExt extends ITelemetryPropertiesExt {
 
 /**
  * Error telemetry event.
- * Maps to category = "error"
+ * @remarks Maps to category = "error"
+ *
+ * @public
  */
 export interface ITelemetryErrorEventExt extends ITelemetryPropertiesExt {
 	eventName: string;
 }
 
 /**
  * Performance telemetry event.
- * Maps to category = "performance"
+ * @remarks Maps to category = "performance"
+ *
+ * @public
  */
 export interface ITelemetryPerformanceEventExt extends ITelemetryGenericEventExt {
 	duration?: number; // Duration of event (optional)
 }
 
 /**
- * An extended TelemetryLogger interface which allows for more lenient event types.
+ * An extended {@link @fluidframework/core-interfaces#ITelemetryBaseLogger} which allows for more lenient event types.
+ *
+ * @remarks
  * This interface is meant to be used internally within the Fluid Framework,
- * and ITelemetryBaseLogger should be used when loggers are passed between layers.
+ * and `ITelemetryBaseLogger` should be used when loggers are passed between layers.
+ *
+ * @public
```

Comment:
Note: this PR is strictly adding explicit annotations to match the existing release type. This item is documented as being intended only for internal use, but it _is_ already published publicly. We cannot make this `@internal` without going through a formal deprecation process.

This PR is strictly intending to add the explicit release tags and make some minor documentation improvements. If we wish to deprecate this and make it internal, that can be done in a subsequent PR. Happy to have a conversation about it here, but resolution is beyond the scope of these changes 🙂

Reply:
this feel wrong to me. what about things already marked internal? currently those are publicly exported?  i think it would be fine to mark this as internal too.

Reply:
i don't think we've ever done the exercise of going through and marking everything that should be internal as internal. its not clear if work like that is planned. if not, i think its even more important to try and tag things correctly, rather what they are today.

Reply:
Hmm. I see your point. I guess my main concern is that I don't want to block rolling out the tag enforcement on long-running conversations about what should and should not be public. And if we didn't already have `@internal` items we need to deal with, I would argue more strongly that we shouldn't make such a change here. But since we'll have to fix other internal items (including ones in this package), it probably does make more sense to just make the tag change now, and add this to the collateral we'll have to deal with when we start deprecating internal exports for external users.

Reply:
Actually, doing so creates a cascade of issues due to other public types using this. If we want to go ahead and mark all of those as internal now, I can. But I would need folks with more context in this space to validate that changes are correct.

Reply:
Related question: should this entire package be internal? Are external customers intended to use these utilities? Or are they just intended to use the base telemetry-related interfaces in `core-interfaces`?

Reply:
Of the top off my head I'd say they're only supposed to use the base interfaces but would need to double check carefully. From the list in the previous comments most things look like they're not for external consumers; I know Loop was using `TelemetryNullLogger` and per our guidance is moving to `createChildLogger`, but I think we missed the fact that it depends on internal interfaces. I think I'd suggest new guidance that the should just create an implementation of a null logger if they need one for their tests, instead of depending on us providing one. Might be an easy-ish PR for us to make to move them away from `createChildLogger()`.

Reply:
I would expect this package to end up all internal.  But a small number of exported members will need to move to core-interfaces or core-utils or whatever.

Reply:
Side note - I just saw how core-interfaces is public (obviously, given what's in it) but core-utils is slated to be internal.  Too bad on the naming, IMO.  I'm wondering where we should put type guards for types exported from core-interfaces...